src/utils/messages.js

```diff
@@ -1,4 +1,4 @@
-const messages ={
+const messages = {
   welcome: 'Welcome to Barefoot Nomad',
   notFound: 'Sorry, we cannot find this endpoint',
```

Comment:
I think this message would sound better if its  ` The route does not exist`

Reply:
I am not the original writer of the message. I only changed the `CRLF` on the line to `LF`. Do you still think I should change it?

Reply:
You may leave it for now since it's just a dummy text